.travis.yml

```diff
@@ -20,4 +20,4 @@ notification:
   - email: false
 
 after_success:
-  - npm run coverage
\ No newline at end of file
+  - npm run coverage
```

Comment:
What is changed here?

Reply:
I was a white space

Reply:
I would suggest to not change anything on this line